README.md

```diff
@@ -33,15 +33,38 @@ The traditional support for the dictionary still works:
 ```swift
 import Foundation
 
-func main(args: [String:Any]) -> [String:Any] {
-    if let name = args["name"] as? String {
+func main(args: Any) -> Any {
```

Comment:
In order to support array result, from the new main signature. we can see, the the input param and outout param should be changed to `Any` both.